### PR TITLE
feat: Adds license endpoints

### PR DESCRIPTION
**Pull request by ahom**

### CODE REVIEW

File: models/LicenseMember.v1.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "description": "Utilisateur attaché à la license Toaztr",
+  "type": "object",
+  "properties": {
+    "ref": {
+      "type": "string",
+      "description": "ID de l'utilisateur."
+    },
+    "name": {
+      "type": "string",
+      "description": "Nom de l'utilisateur."
```
Comment (Coleim) [inline models/LicenseMember.v1.json:12]: On rajoute le mail ?

File: models/SubscriptionType.v1.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "string",
+  "title": "SubscriptionType",
+  "enum": [
+    "SOLO",
+    "TEAM"
```
Comment (Coleim) [inline models/SubscriptionType.v1.json:6]: Tu rajoutes pas entreprise ? Ou on verra apres.

**Review by Coleim — COMMENTED** [review]
Du coup on fait 1 suscription <=> 1 licence
On a pas les multi users pour le moment ?

File: models/LicenseMember.v1.json (hunk 1 above)
Comment (mvicrob) [reply models/LicenseMember.v1.json:12]: ++

File: models/SubscriptionType.v1.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "string",
+  "title": "SubscriptionType",
+  "enum": [
+    "SOLO",
```
Comment (mvicrob) [inline models/SubscriptionType.v1.json:5]: SOLO looks frenchy no? I would put INDIVIDUAL, no ?

File: reference/toaztr.v1.yaml [hunk 4]
```diff
@@ -1555,6 +1566,58 @@ paths:
...
+          $ref: '#/components/responses/500'
+      summary: Retourne la license en cours de l'utilisateur
+      security:
+        - toaztr_auth:
+          - infos
+      tags:
+        - license
+  /v1/license/members:
+    get:
+      summary: Liste les membres associés à la license de l'utilisteur
```
Comment (mvicrob) [inline reference/toaztr.v1.yaml:1596]: utilisateur

File: models/SubscriptionType.v1.json (hunk 2 above)
Comment (ahom) [reply models/SubscriptionType.v1.json:6]: Je comptais le faire plus tard oui, car le team n'est meme pas fini de toute facon

File: models/SubscriptionType.v1.json (hunk 3 above)
Comment (ahom) [reply models/SubscriptionType.v1.json:5]: We can change the name if you guys prefer, but I don't understand why it looks frenchy?

File: models/LicenseMember.v1.json (hunk 1 above)
Comment (ahom) [reply models/LicenseMember.v1.json:12]: This is just reflecting what is in SubscriptionItems which do not include the email.
Why should we store the mail here? I don't see the usecase

File: models/SubscriptionType.v1.json (hunk 3 above)
Comment (Coleim) [reply models/SubscriptionType.v1.json:5]: It has moustaches

**Review by Coleim — APPROVED** [review]

**Pull request merged**